common-rest/src/test/java/org/uniprot/api/rest/validation/ValidDownloadRequestTest.java

```diff
@@ -9,9 +9,21 @@
 class ValidDownloadRequestTest {
 
     @Test
-    void when_fields_passed_then_invalid_request() {
+    void when_fields_passed_with_supported_format_then_valid_request() {
         FakeDownloadRequest downloadRequest = new FakeDownloadRequest();
-        downloadRequest.setFormat(ValidDownloadRequest.FORMATS_WITH_NO_PROJECTION.get(0));
+        downloadRequest.setFormat("json");
+        downloadRequest.setFields("field1,field2");
+        FakeDownloadRequestValidator validator = new FakeDownloadRequestValidator();
+        boolean result = validator.isValid(downloadRequest, null);
+        assertTrue(result);
+        assertNull(validator.errorMessage);
+    }
+
+    @Test
+    void when_fields_passed_with_unsupported_format_then_invalid_request() {
+        FakeDownloadRequest downloadRequest = new FakeDownloadRequest();
+        downloadRequest.setFormat(
+                ValidDownloadRequest.FORMATS_WITH_NO_PROJECTION.stream().findFirst().get());
```

Comment:
iterator().next() could be a simpler way here.

Reply:
agree.